### PR TITLE
chore: enable cypress integration tests

### LINKED ISSUE #2451: Enable Cypress integrations tests
This disabled in #2443 due to this error:

> You've exceeded the limit of private test recordings under your free plan this month. The limit is 500 private test recordings.
To continue recording tests this month you must upgrade your account. Please visit your billing to upgrade to another billing plan.

### PR DESCRIPTION
**Pull request by HashimotoYT**
Fixes #2451

### CODE REVIEW

**Comment by malept** [comment]
I wonder if the cypress tests should be in their own [build stage](https://docs.travis-ci.com/user/build-stages/).

**Comment by HashimotoYT** [comment]
It's not give speed, but add some readability. We still around the 5-10 per build as per, it's very big time for such small website.

File: .travis.yml [hunk 1]
```diff
@@ -6,5 +6,10 @@ node_js:
 notifications:
   email: false
 
-script:
-  - npm test && npm run integration
+jobs:
+  inclide:
```
Comment (malept) [inline .travis.yml]:
```suggestion
  include:
```

File: .travis.yml [hunk 2]
```diff
@@ -6,5 +6,10 @@ node_js:
...
+    - stage: "Tests"
+      name: "Unit tests"
+      scirpt: npm test
```
Comment (malept) [inline .travis.yml]:
```suggestion
      script: npm test
```

**Pull request merged**